examples/cua/legacy_report_fetcher/run_demo.py

```
@@ -0,0 +1,786 @@
+        "report_date": report_date,
+        "status": "success",
+        "files": files,
+    }
+    manifest_path = inbox / "manifest.json"
+    manifest_path.write_text(json.dumps(manifest, indent=2) + "\n", encoding="utf-8")
+    return inbox
+
+
+def run_pipeline(inbox: Path) -> str:
+    """Run the existing-pipeline stand-in on the host handoff folder."""
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)